### PR TITLE
Add script for getting new TLS certificates

### PR DESCRIPTION
**Pull request by dshiga**
To make this easier next time we need new certificates, I created a Dockerfile and a script to build and run it, which automates the process of obtaining new certificates.

See https://elastc.com/c/SyXB5BII/405-get-new-ssl-certificates

### CODE REVIEW

File: certs/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+For example:
+```
+export AWS_ACCESS_KEY_ID=foo
+export AWS_SECRET_ACCESS_KEY=bar
+```
+
+Usage
+-----
+bash get_certs.sh work_dir domain
```
Comment (rexwangcc) [inline certs/README.md]: Would be better to use: `bash get_certs.sh work_dir domain` here

File: certs/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+where work_dir is the directory you'd like certificate to be written to
+and domain is the domain you'd like to obtain a certificate for
+
+Example
+-------
+bash get_certs.sh $(pwd) pipelines.dev.data.humancellatlas.org
```
Comment (rexwangcc) [inline certs/README.md]: `bash get_certs.sh $(pwd) pipelines.dev.data.humancellatlas.org`

File: certs/get_certs.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+working=$1
+domain=$2
+
```
Comment (rexwangcc) [inline certs/get_certs.sh]: I'd add something like:
```
tag=$1
if [ -z "$working" ]; then
  echo "Must pass in the working directory to use for docker image, e.g. $(pwd)"
  exit 1
fi

if [ -z "$domain" ]; then
  echo "Must pass in the domain to use for docker image, e.g. pipelines.dev.data.humancellatlas.org"
  exit 1
fi
```
So if we miss some inputs, it won't throw the error after building the docker. What do you think?

File: certs/README.md (hunk 1 above)
Comment (dshiga) [reply certs/README.md]: I'm confused -- how is that different from what the README says?

Comment (dshiga) [reply certs/README.md]: Oh I see, you'd like backticks to escape it as code, will do.

Comment (rexwangcc) [reply certs/README.md]: Sorry I didn't make it clear, yeah adding backticks is what I mean.

**Pull request merged**